src/deployers/BasicPluginsDeployer.cpp

```diff
@@ -38,8 +38,13 @@ bool BasicPluginsDeployer::deployStandardQtPlugins(const std::vector<std::string
 {
     for (const auto &pluginName : plugins) {
         ldLog() << "Deploying Qt" << pluginName << "plugins" << std::endl;
+        fs::path pluginSubDir = pluginName;
+        // add a trailing slash, so it is used as a destination directory, not a file.
+        if (pluginSubDir.has_filename()) {
+            pluginSubDir += "/";
+        }
         for (fs::directory_iterator i(qtPluginsPath / pluginName); i != fs::directory_iterator(); ++i) {
-            if (!appDir.deployLibrary(*i, appDir.path() / "usr/plugins" / pluginName))
+            if (!appDir.deployLibrary(*i, appDir.path() / "usr/plugins" / pluginSubDir))
```

Comment:
Wouldn't it be easier to do the following?

```suggestion
            // add a trailing slash, so it is used as a destination directory, not a file.
            if (!appDir.deployLibrary(*i, appDir.path() / "usr/plugins" / pluginName / ""))
```

Reply:
Yes, that is shorter and will also work. Quick check, all as expected:
```
    std::cout << fs::path("foo") / "bar" / "" << '\n';
    std::cout << fs::path("foo") / "bar/" / "" << '\n';
    fs::path d = fs::path("foo") / "bar" / "";
    std::cout <<  d.string().back() << '\n';     // from appdir.cpp:453
```
->
```
"foo/bar/"
"foo/bar/"
/

```